buildbot-worker/Dockerfile

```diff
@@ -22,6 +22,12 @@ RUN Write-Host 'Installing buildbot ...'; `
 
 COPY BuildbotWorker C:/BW
 
+#--------------------------------------------------------------------
+# Make git sh available in path for branch cleanup
+#--------------------------------------------------------------------
+
+RUN Register-SystemPath -path 'C:\\program files\\git\\bin'
```

Comment:
You shouldn't need to quote it. In the tools Dockerfile there's

```
RUN Register-SystemPath -Path C:\Tools\diffutils\bin; `
    diff3 --version;
```

Is there anything you can do that'll print a version to make sure whatever you're after is available?

Reply:
It seems to be unhappy with path names with spaces in them without quoting for me.

Reply:
Oh sorry my bad there's spaces in the path.

Reply:
I'm adding an `sh --version` to the line to get the version info now.